src/main/java/org/folio/innreach/domain/service/impl/CentralPatronTypeMappingServiceImpl.java

```diff
@@ -33,7 +34,7 @@
   public CentralPatronTypeMappingsDTO getCentralPatronTypeMappings(UUID centralServerId, Integer offset, Integer limit) {
     var centralPatronTypeMappingExample = mappingExampleWithServerId(centralServerId);
 
-    var centralPatronTypeMappings = repository.findAll(centralPatronTypeMappingExample, PageRequest.of(offset, limit));
+    var centralPatronTypeMappings = repository.findAll(centralPatronTypeMappingExample, new OffsetRequest(offset, limit, Sort.unsorted()));
```

Comment:
Default sorting should be used instead of `Sort.unsorted()` in all places to get predictable and repeatable results